modnet/preprocessing.py

```diff
@@ -128,43 +144,65 @@ def get_cross_nmi(df_feat: pd.DataFrame, drop_thr: float = 0.2, return_entropy =
     else:
         seed = np.random.RandomState()
 
+    if kwargs.get("n_neighbors"):
+        n_neighbors = kwargs.pop("n_neighbors")
+    else:
+        n_neighbors = 3
+
     # Prepare the output DataFrame and compute the mutual information
     mutual_info = pd.DataFrame([], columns=df_feat.columns, index=df_feat.columns)
 
+    #create pool of workers
+    pool = Pool(processes=cpu_count())
```

Comment:
This should be configurable, and all CPUs (probably x2 with hyper-threading) is not a sensible default

Reply:
This is now configurable and defaults to 1, for now. I think I will refactor the pool slightly in a future PR so that it uses a context manager to safely close/join the pool.